src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
 <h1>NgWig Demo</h1>
-<ng2wig></ng2wig>
+<ng2wig plugins="demo"></ng2wig>
```

Comment:
why do we need to change HTML here?

I'm thinking about configuring the module, to make it possible to run ngWig with extensions, like:

    imports: [
      ...
      Ng2WigModule.forRoot({extentions: [MyPluginModule/MyPluginComponent]})
      ...
    ],